app/views/compute_resources_vms/form/_openstack.html.erb

```diff
@@ -12,7 +12,7 @@
 </div>
 <%= select_f f, :tenant_id, compute_resource.tenants, :id, :name, {}, :label => _('Tenant') %>
 
-<%= select_f f, :security_groups, compute_resource.security_groups, :name, :name, {}, :label => _("Security group")  %>
+<%= select_f f, :security_group, compute_resource.security_groups, :name, :name, {}, :label => _("Security group")  %>
```

Comment:
@ohadlevy 

It was security_group in https://github.com/theforeman/foreman/commit/88daf556e39ad9c0d3bdebbe9f88ce5ff49010f6#diff-e8ad247c98e1f7a4c16240624c756059, 
and then changed to `security_groups` in https://github.com/theforeman/foreman/commit/e972c0e6740e92e3cc595ab14d4fb6a2567aa1f5#diff-7d5ab47e2462b7abd18eeb88b504fee0.

Why? 

It looks like `security_group` has something to do with the extensions to Fog we've added... https://github.com/theforeman/foreman/blob/develop/app/models/concerns/fog_extensions/openstack/server.rb.